src/validation.cpp

```diff
@@ -814,7 +814,6 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
         return false; // state filled in by CheckTxInputs
     }
 
-    // Check for non-standard pay-to-script-hash in inputs
```

Comment:
Could we just make this `// Check for non-standard inputs` instead of removing entirely? Just for consistency like below for witnesses. I suppose `AreInputsStandard()` is self-documented anyway so I can understand the want to remove it.

Reply:
I'm fine with removing it in this case. In contrast to a lot of code "fRequireStandard && !AreInputsStandard(tx, m_view)" looks very self-documenting to me, maybe "Check for non-standard inputs" wouldn't add anything?